package.json

```diff
@@ -18,34 +19,37 @@
   "author": "PEX",
   "license": "MIT",
   "engines": {
-    "node": ">=14"
+    "node": "20.x || 22.x"
   },
   "bugs": {
     "url": "https://github.com/HMPO/hmpo-model/issues"
   },
   "homepage": "https://github.com/HMPO/hmpo-model#readme",
   "dependencies": {
-    "debug": "^4.3.6",
+    "debug": "^4.3.7",
     "got": "<12",
     "http-proxy-agent": "^5.0.0",
     "https-proxy-agent": "^5.0.1",
```

Comment:
There's 7.x versions available for both of these now which we should upgrade to if we can